apps/omg_watcher/test/omg_watcher/db/txoutput_test.exs

```diff
@@ -52,7 +52,7 @@ defmodule OMG.Watcher.DB.TxOutputTest do
     power_of_2 = fn n -> :lists.duplicate(n, 2) |> Enum.reduce(&(&1 * &2)) end
     assert 16 == power_of_2.(4)
 
-    big_amount = power_of_2.(260)
+    big_amount = power_of_2.(256) - 1
```

Comment:
why this change :) ?

Reply:
Amount is encoded as UInt256 before hashing :)